Libplanet.Net.Tests/Consensus/ConsensusContext/ConsensusContextTest.cs

```diff
@@ -313,5 +313,35 @@ void CatchPropose(object? sender, ConsensusMessage? message)
             Assert.NotNull(blockChain.Store.GetLastCommit(blockChain.Tip.Index));
             Assert.Null(blockChain.Store.GetLastCommit(blockChain.Tip.Index - 1));
         }
+
+        [Fact(Timeout = Timeout)]
+        public void RemoveOldContexts()
+        {
+            var validators = new List<PublicKey>
+            {
+                TestUtils.Peer0Priv.PublicKey, TestUtils.Peer1Priv.PublicKey,
+            };
+            var (fx, blockChain, consensusContext) = TestUtils.CreateDummyConsensusContext(
+                TimeSpan.FromSeconds(1),
+                TestUtils.Policy,
+                TestUtils.Peer1Priv,
+                validators,
+                lastCommitClearThreshold: 1);
+
+            // Create context of index 1.
+            consensusContext.NewHeight(1);
+            // Create context of index 2.
+            consensusContext.HandleMessage(
+                new ConsensusPropose(validators[1], 2, 1, fx.Hash1, new byte[] { }, -1));
+
+            blockChain.Append(blockChain.ProposeBlock(new PrivateKey()));
+            blockChain.Append(blockChain.ProposeBlock(new PrivateKey()));
+            blockChain.Append(blockChain.ProposeBlock(new PrivateKey()));
```

Comment:
```suggestion
            blockChain.Append(blockChain.ProposeBlock(new PrivateKey()));
            blockChain.Append(blockChain.ProposeBlock(new PrivateKey(), TestUtils.CreateLastCommit(blockChain.Tip.Hash, 2, 0))));
            blockChain.Append(blockChain.ProposeBlock(new PrivateKey(), TestUtils.CreateLastCommit(blockChain.Tip.Hash, 3, 0))));
```

Tests are failing due to `LastCommit`.